content/en/docs/concepts/project-maturity.md

```diff
@@ -31,5 +27,5 @@ many versions as possible, it is sometimes required to lose compatibility in
 the interest of furthering the feature set of cert-manager and making use of
 newer features available in upstream Kubernetes.
 
-As of cert-manager version `v0.11`, the lowest Kubernetes version supported is
-`v1.12`.
+As of cert-manager version `v1`, the lowest Kubernetes version supported is
```

Comment:
I think "As of v0.11" still holds here

Reply:
Yeah, I agree. But perhaps I should just remove this paragraph, because it contradicts the earlier definition of supported kubernetes versions.

Reply:
it is, we added 1.11 back thanks to filter-crd

Reply:
Looks like filter-crd was merged in https://github.com/jetstack/cert-manager/commit/8152159ea764b5180ace01904b5e742d7a0bbb13 and first used in cm 1.14 so I've updated the doc accordingly.